controllers/authControllers.js

```diff
@@ -3,29 +3,33 @@ const db = require('../db/authHelpers');
 module.exports.registerAdmin = function (req, res) {
     try {
         const data = req.body
-        const {username, password, email, organisation}=data
+        const { username, password, email, organisation } = data
```

Comment:
Validate each data coming from the client. (use any of  library like express-validator , validator , chai js or joi)